src/error.cpp

```diff
@@ -64,6 +64,7 @@ const char *err_str(Error err) {
         case ErrorInvalidCpuFeatures: return "invalid CPU features";
         case ErrorInvalidLlvmCpuFeaturesFormat: return "invalid LLVM CPU features format";
         case ErrorUnknownApplicationBinaryInterface: return "unknown application binary interface";
+        case ErrorASTUnitFailure: return "ASTUnit failure";
```

Comment:
These are potentially user-facing error messages, I would prefer something like this (to save us from answering questions on IRC): 

```suggestion
        case ErrorASTUnitFailure: return "compiler bug: clang encountered a compile error, but the libclang API does not expose the error. See https://github.com/ziglang/zig/issues/4455 for more details.";
```

(I'll make this change in master after I have cleared up some dirty working trees in git)